sdk/python/kfp/_client.py

```diff
@@ -474,6 +474,17 @@ def schedule_pipeline(self, experiment_id, job_name, pipeline_package_path=None,
     #[TODO] Add link to the scheduled job. 
     response = self._job_api.create_job(body=schedule_body)
 
+  def list_jobs(self, page_token='', page_size=10, sort_by=''):
```

Comment:
Why do you pass these arguments? I suggest the following: 
```suggestion
  def list_jobs(self):
```

Reply:
the api have them as input parameters see https://github.com/kubeflow/pipelines/blob/master/backend/api/swagger/job.swagger.json Thought it would be nice to let the user specify it if wanted, and that they exsist for a lot of the other functions.

Reply:
In order for them to be relevant you need to pass it to: 

```python 
self._job_api.list_jobs(page_token=page_token, page_size=page_size, sort_by=sort_by)
```

Here you can see an example: https://github.com/kubeflow/pipelines/blob/7e610d1a3298891c76e76777560068ecfac56b36/sdk/python/kfp/_client.py#L489

I also believe you don't want to have the defaults, they might have a influence that are unwanted